Havit.Blazor.Components.Web.Bootstrap/Forms/HxInputRange.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components.Forms;
+
+namespace Havit.Blazor.Components.Web.Bootstrap;
+
+/// <summary>
+/// Allows the user to select a number in a specified range using a slider.
+/// </summary>
+public class HxInputRange : HxInputBase<float>
```

Comment:
@hakenr I consulted the component with @jirikanda, and we came to a conclusion to drop `Min`, `Max`, and `Step` from the settings so that they don't have to be generic - I find having `Step` in Defaults (Settings) to be beneficial, do you think adding it to the settings as a `decimal` (for example) would make sense?

Also, I added a constraint for `TValue` to be a `non-nullable value type` because `HxInputRange` supports most of them and the `Min`, `Max`, and `Step` parameters could be made `nullable`, so they don't have to be set and defaults from Bootstrap are taken. Otherwise, it would be challenging to supply default values to the generic parameters (users would be required to always set them).

Reply:
- [ ] remove the `where TValue : struct` constraint and implement the full check for supported types in constructor as `HxInputNumber` does (we can check the non-nullability there)
- [ ] use `TValue` (instead of `TValueType` for the demos)